main/autocode/src/org/ejml/GenerateJavaCode32.java

```diff
@@ -53,7 +53,9 @@ public GenerateJavaCode32() {
         prefix64.add("DUnary");
         prefix32.add("FUnary");
         prefix64.add("DBinary");
+        prefix64.add("IDBinary");
```

Comment:
Can you keep the ordering of 64 then 32 name? easier to keep track of what is converted into what.

Reply:
Sure :+1:

Reply:
Looking at it now, I really don't know, how it got there...